### PR TITLE
Reverse geocoding.

### PR DESCRIPTION
**Pull request by moorchegue**
Addresses #32

### CODE REVIEW

File: services/geocoding.py [hunk 1]
```diff
@@ -70,4 +76,41 @@ def enhance(self, entries: List[PointOfInterest]) -> List[PointOfInterest]:
...
+        if not entry.country:
+            entry.country = next(
+                r["long_name"] for r in geodata
+                if "country" in r["types"])
+
+        if not entry.city:
+            city_levels = {
+                "administrative_area_level_1",
+                "administrative_area_level_2",
+            }
```
Comment (idisblueflash) [inline services/geocoding.py:108]: why not use `locality` for the city?
Screenshot 2022-06-12 at 19 42 54

**Review by littlepea — APPROVED** [review]

File: services/geocoding.py (hunk 1 above)
Comment (moorchegue) [reply services/geocoding.py:108]: Actually I'm not sure what to use, some answers do have locality, some don't, some adm levels 1 or 2 seem appropriate… I guess I'll add all three in order of increasing significance.

**Comment by moorchegue** [comment]
Hey @idisblueflash, I just rebased against latest master, and looks like that broje some tests. Could you take a peek, please?

**Comment by littlepea** [comment]
@idisblueflash yes, you've committed directly to master without running tests it seems...

**Comment by moorchegue** [comment]
I'll merge this first then. The rest of the tests are passing.

**Comment by idisblueflash** [comment]
Oh, That's my bad.

I'll fix it tomorrow morning(about 9 hours later)

**Comment by idisblueflash** [comment]
674dfa01ab8505926c2262872b44885784dbd40a

**Pull request merged**